agent/status.go

```diff
@@ -35,6 +35,7 @@ func (a *Agent) startStatusServer(ctx context.Context, port int) error {
 
 	app.AddMiddleware(gimlet.MakeRecoveryLogger())
 	app.AddRoute("/status").Handler(a.statusHandler()).Get()
+	app.AddRoute("/metadata_tag").Handler(a.addMetadataTagHandler).Post()
```

Comment:
nit: might be better to call it `/failure_metadata_tag` so that it's consistent throughout the documentation with the field.